backend/app/services/subscription/follow_service.py

```
@@ -294,6 +298,84 @@ def get_followers_count(
+            db: Database session
+            subscription_id: ID of the subscription
+
+        Returns:
+            List of follow records with notification enabled
+        """
+        return (
+            db.query(SubscriptionFollow)
+            .filter(
+                SubscriptionFollow.subscription_id == subscription_id,
+                SubscriptionFollow.invitation_status == InvitationStatus.ACCEPTED.value,
+                SubscriptionFollow.enable_notification == True,
```

Ruff (0.14.14)

[error] Avoid equality comparisons to `True`; use `SubscriptionFollow.enable_notification:` for truth checks

Replace with `SubscriptionFollow.enable_notification`

(E712)